### ISSUE TITLE
Define a Simple Data Model and OData Service with CDS

### ISSUE THREAD

**Issue by BaerbelW** [SAPDocuments/Tutorials#4111]
Tutorial URL: https://developers.sap.com/tutorials/odata-05-data-model-service.html

Please specify the step you are referring to
Step 3: Set CF and CDS preferences in SAP Web IDE

The screen I get when I check the Cloud Foundry preferences looks different to what is shown in the tutorial as there isn't anything pre-filled in the input fields and when I select the shown API endpoint I get a pop-up to login:
![grafik](https://user-images.githubusercontent.com/18588108/67576741-6a4de180-f73f-11e9-84d9-9560837065c7.png)

Is this step still needed? The informational message at the top seems to indicate that it's no longer needed.

I had worked through the tutorial shown as a prerequisite.

Thanks and Cheers
Bärbel

**Comment by BaerbelW** [comment]
Not too surprisingly, not being able to complete Step 3 to select the CF Space, I'm now not able to complete Step 11 to create the database. When I try to build the DB I get an error message:
![grafik](https://user-images.githubusercontent.com/18588108/67671624-969f7300-f976-11e9-9494-3b94eae1acd6.png)

Being a complete beginner and mostly not really knowing what I'm doing where and why in the course of the tutorial, I'm not really sure how the mentioned prerequisite for the Cloud Foundry mentioned in the linked [official help documentation](https://help.sap.com/viewer/825270ffffe74d9f988a0f0066ad59f0/CF/en-US/98f49286ac05492f88428c603d146fc3.html) applies to the trial setup. How could I e.g. "ask your administrator to create a subaccount for you" in this type of setting as mentioned in the help documentation?

What could I try/do?

Thanks & Cheers
Bärbel

**Comment by qmacro** [comment]
Hi @BaerbelW  - I would suggest a two level approach. If we can get you going quickly here, let's do that; otherwise I'd suggest you take this to answers.sap.com as it's not specifically related to the tutorial content itself, and others may benefit from the question and the answer. 

Let's see if we can get you going quickly - go to https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trial and hit the "Enter Your Trial Account" button. Follow the process for creating the subaccount, CF org and space, and see where you get to. Then retry this step in the tutorial. If you get stuck with this, come back to me here first :-)

In the meantime, I'll be modifying the step (3) anyway thanks to an issue that @IObert pointed out #3730

**Comment by BaerbelW** [comment]
Hi DJ - thanks for bearing with me. I'm really confused by all of this (I'm sure you can tell!) and it doesn't help that I don't really know what I'm doing or supposed to be doing here. So, I think I now have a CF trial account:
![grafik](https://user-images.githubusercontent.com/18588108/68400258-0502e300-0178-11ea-9887-da0bb02306f5.png)

Do I now have to click on "New Subaccount" and if yes, what - exactly - do I have to enter/select in the next pop-up?
![grafik](https://user-images.githubusercontent.com/18588108/68400389-409dad00-0178-11ea-86bd-d6714728e0a3.png)

Thanks much & Cheers
Bärbel

**Comment by qmacro** [comment]
Just click on the big while oblong (with "trial" in the top left of it) - that's your existing subaccount. That should take you to where you can check / create your CF org and space. I think at this stage it's better if you take this over to answers.sap.com as we're moving away from specifics for this particular tutorial - hope that's OK! See you over at answers!

**Comment by BaerbelW** [comment]
Hi DJ,

all sorted (I think): while capturing screenshots for a question to ask on answers, I tried to login with another combination of email & pw I saw in Chrome's list of stored credentials which was accepted so I could then proceed. While in the list I also deleted some entries utilising my S-accounts instead of the P-account I had set up for the trial. It's really tricky to keep track of all these different logins needed to work through just a simple tuturoial!

Thanks for helping!

Cheers
Bärbel
